main.py

```diff
@@ -41,15 +36,4 @@
                   "MFC CORRECTION",
                   "DATE STAMP"]
 
-    with open("data/norway_welldb.csv", 'w') as file:
-        file.write(",".join(attributes) + "\n")
-        for dataframe in log_headers:
-            extract_attributes = dataframe.loc[:, attributes]
-
-            # Finally, only include logs whose name contains "STAT"
-            filtered_log_headers = extract_attributes.loc[extract_attributes["NAME"].str.contains("STAT")]
-
-            # Print first three log entries:
-            print(filtered_log_headers.loc[0:3, :])
-
-            filtered_log_headers.to_csv(file, mode='a', index=False, header=False)
+    _ = write_log_headers_to_file(attributes=attributes, project=project, file_path="data/test.csv")
```

Comment:
```suggestion
    write_log_headers_to_file(attributes=attributes, project=project, file_path="data/test.csv")
```

do we need _ = ?
looks like write_log_headers_to_file does not return anything

Reply:
True! Thanks 😄